src/main/java/com/codessquad/qna/question/Question.java

```diff
@@ -11,6 +11,8 @@
 @Entity
 public class Question {
 
+  private static DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm");
```

Comment:
```suggestion
  private static final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm");
```

변경하지 않고 사용할거라면, final을 붙여주는것도 좋을 것 같네요